libs/prelude/Prelude/Interactive.idr

```diff
@@ -89,6 +89,16 @@ putCharLn c = putStrLn (singleton c)
 getChar : IO Char
 getChar = map cast $ foreign FFI_C "getchar" (IO Int)
 
+||| Disables buffering in both stdin and stdout
+||| so that output is written immediately, and never stored in the buffer
+||| and the next input item is read and returned
+|||
+||| this is useful to circumvent problem with IO on some windows system
```

Comment:
Could you kindly replace 'windows system' with 'Windows systems'? (I think capitalization helps realising that it's about the operating system).